doc/nrf/known_issues.rst

```diff
@@ -161,6 +161,14 @@ NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
 Other issues
 ============
 
+.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-14312: During execution, in rare cases, the :c:func:`nrf_recv()` function would crash.
+
+.. rst-class:: v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-13360: When a socket that is in a :c:func:`nrf_recv()` loop is closed by another thread, it will eventually access the packet handler in the socket context causing a crash since the socket context was destroyed by the :c:func:`nrf_close()` call.
```

Comment:
```suggestion
.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

NCSDK-14312: The :c:func:`nrf_recv` function crashes
  During execution, in rare cases, the :c:func:`nrf_recv` function crashes  in the :ref:`nrfxlib:nrf_modem`.

.. rst-class:: v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0

NCSDK-13360: The socket context crashes when closed by another thread
  When a socket that is in a :c:func:`nrf_recv` function loop is closed by another thread, the socket eventually accesses the packet handler in the socket context, causing a crash.
  This is because the socket context was destroyed by the :c:func:`nrf_close` function call.
```
1. We could move both the issues to nrfxlib, create a new section for "Modem Library" after the "GNSS sockets" header.
2. Just re-confirming whether we need to add v1-6 because the issue will now be fixed in this ncs version?

Reply:
There is no tag on the v1.6-branch where these bugs are fixed (yet?). Both v1.6.0 and v1.6.1 have this bug.